packages/components/socialFeed/SocialActions/TipButton.tsx

```diff
@@ -113,17 +113,21 @@ export const TipButton: React.FC<{
               amount={tipAmountLocal}
             />
           ) : (
-            <TeritoriTipAmount amount={amount} />
+            <TeritoriTipAmount amount={tipAmountLocal} />
           )}
         </BrandText>
       </TouchableOpacity>
 
       <TipModal
         authorId={authorId}
         postId={postId}
-        onClose={(newTipAmount: number | undefined) => {
+        onClose={(addedTipAmount: number | undefined) => {
           setTipModalVisible(false);
-          newTipAmount && setTipAmountLocal(amount + newTipAmount);
+          addedTipAmount &&
+            setTipAmountLocal(
+              (typeof amount !== "number" ? parseInt(amount, 10) : amount) +
+                addedTipAmount,
+            );
```

Comment:
please don't add a hack and find root cause of why a string is passed

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1387/commits/db6a08fb1ffd5ea9da43bfa210d9e35f2e4e9207